### PR TITLE
[IoTHub] Add missing parameter to create storage endpoint

### PR DESCRIPTION
**Pull request by anusapan**
---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [x] The PR has modified HISTORY.rst describing any customer-facing, functional changes. Note that this does not include changes only to help content. (see [Modifying change log](https://github.com/Azure/azure-cli/tree/master/doc/authoring_command_modules#modify-change-log)).

- [x] I adhere to the [Command Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/command_guidelines.md).

### CODE REVIEW

**Comment by fengzhou-msft** [comment]
@Juliehzl please take a look.

File: src/azure-cli/azure/cli/command_modules/iot/_params.py [hunk 1]
```diff
@@ -203,6 +203,17 @@ def load_arguments(self, _):  # pylint: disable=too-many-statements
                    help='Encoding format for the container. The default is AVRO. '
                         'Note that this field is applicable only for blob container endpoints.')
 
+    with self.argument_context('iot hub routing-endpoint create') as c:
+        c.argument('batch_frequency', options_list=['--batch-frequency', '-b'], type=int,
```
Comment (Juliehzl) [inline src/azure-cli/azure/cli/command_modules/iot/_params.py]: Can you specify unit for your parameter in help message? And from my understanding, the value must be in [60, 720], right?

File: src/azure-cli/azure/cli/command_modules/iot/_params.py [hunk 2]
```diff
@@ -203,6 +203,17 @@ def load_arguments(self, _):  # pylint: disable=too-many-statements
...
+                   help='The maximum amount of time that can elapse before data is'
+                        ' written to a blob, between 60 and 720 seconds.')
+        c.argument('chunk_size_window', options_list=['--chunk-size', '-w'], type=int,
```
Comment (Juliehzl) [inline src/azure-cli/azure/cli/command_modules/iot/_params.py]: The same as above. Please specify unit here.

**Comment by Juliehzl** [comment]
LGTM in general.

File: src/azure-cli/azure/cli/command_modules/iot/_params.py (hunk 1 above)
Comment (anusapan) [reply src/azure-cli/azure/cli/command_modules/iot/_params.py]: Updated

File: src/azure-cli/azure/cli/command_modules/iot/_params.py (hunk 2 above)
Comment (anusapan) [reply src/azure-cli/azure/cli/command_modules/iot/_params.py]: Updated

**Pull request merged**